src/components/OptionButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { type ReactNode } from "react";
```

Comment:
consider exporting related components under an `index.ts`

Take a look at how its done under `components/booking/SelectSessionList` etc.

This can be done in multiple places, such as with your SemesterComponents.

You can also consider using the compound component pattern in cases where you think it makes sense.
https://medium.com/@vitorbritto/react-design-patterns-compound-component-pattern-ec247f491294#:~:text=Compound%20components%20are%20a%20pattern,state%20and%20handle%20logic%20together.

I think that the Popover component can adopt this pattern:

```
<Popover>
 <Popover.Button>
    ...
 </Popover.Button>
 <Popover.Container>
    ...
 </Popover.Container>
</Popover>
```